### DISCUSSION TITLE
couple of question

### DISCUSSION THREAD

**Discussion by JunkyardCat** [arianneorpilla/jidoujisho#169] [General]
Whats the web browser you use in uour app? Is it kiwi?

**Comment by arianneorpilla** [comment]
Android WebView

**Comment by JunkyardCat** [comment]
Is it possible to use this android web view to view a website with yomichan support?

**Reply by arianneorpilla** [reply]
Yomichan works via Mozilla/Google's Extension Manifest/API. It is not possible to inject Yomichan as is into a WebView (without thorough forking and code changes).

My app works by injecting JavaScript to allow me to communicate between the WebView and my application. This bridge merely grabs selected text and modifies websites to my liking, and passes back messages that my app can use to perform its own native functionality, with my own dictionary system and handling of media.

If you merely mean if I can allow you to use a browser with an overlay similar to how I implement TTSU/Mokuro built-in support, that is possible in the future.

As much as I would like to accomplish a lot on my own terms (as I enjoy maintaining my own workflow), in the meantime I have to refer you to a Kiwi/Yomichan workflow. I am able to tightly knit my featureset with TTSU/Mokuro but the support that I have is due to these having websites that I can very specifically control the content of. If it is a general website that could have anything in it, a browser-first solution would be best.

**Reply by JunkyardCat** [reply]
Thanks for the reply. I havemt fully understood what you said but i think i understand the gist of it that it cant be done.
